packages/devtools_app/lib/src/service/vm_service_wrapper.dart

```diff
@@ -502,17 +502,17 @@ class VmServiceWrapper implements VmService {
       supportedVersion: SemanticVersion(major: 1, minor: 3),
       isolateId: isolateId,
     )) {
-      return isolate.extensionRPCs
+      return isolate.extensionRPCs!
```

Comment:
to make this a bit more null safe, add a variable after `isolate` is defined and use that for both these checks

`final extensionRPCs = isolate.extensionRPCs ?? <String>[]`

Reply:
done